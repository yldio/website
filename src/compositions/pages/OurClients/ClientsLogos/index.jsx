import React from 'react';
import PageSection from 'components/PageSection';
import { Grid, Header, Line, LogoWrapper, Image } from './styled';
import Hive from '../../../../assets/hive.png';
import Economist from '../../../../assets/economist.png';
import Tate from '../../../../assets/tate.png';
import Discovery from '../../../../assets/discovery.png';
import Canon from '../../../../assets/canon.png';
import SamsungJoyen from '../../../../assets/samsung-joyent.png';
import Kingfisher from '../../../../assets/kingfisher.png';
import Trainline from '../../../../assets/trainline.png';
import Conde from '../../../../assets/conde-nas.png';
import Microsoft from '../../../../assets/microsoft.png';
import Newsuk from '../../../../assets/newsuk.png';
import Google from '../../../../assets/google.png';
import Thomascook from '../../../../assets/thomascookLogo.png';
import Compare from '../../../../assets/compare-market.png';
import Solera from '../../../../assets/solera.png';
import Skype from '../../../../assets/skype.png';
import Tes from '../../../../assets/tes.png';
import Perform from '../../../../assets/perform.png';
import Lamassu from '../../../../assets/lamassu.png';

const logos = [
  Hive,
  Economist,
  Tate,
  Discovery,
  Canon,
  SamsungJoyen,
  Kingfisher,
  Trainline,
  Conde,
  Microsoft,
  Newsuk,
  Google,
  Thomascook,
  Compare,
  Solera,
  Skype,
  Tes,
  Lamassu,
  Perform,
];

const ClientsLogos = index => (
  <PageSection>
    <Header>Who we work with</Header>
    <Line />
    <Grid>
      {logos.map(logo => (
        <LogoWrapper>
          <Image alt="" key={index} src={logo} />
        </LogoWrapper>
      ))}
    </Grid>
  </PageSection>
);

export default ClientsLogos;
