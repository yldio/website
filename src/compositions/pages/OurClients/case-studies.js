import TrainlineScreenshots from '../../../assets/trainline-screen.png';
import TheEconomistScreenshots from '../../../assets/economist-screens.png';
import ThomasCookScreenshots from '../../../assets/thomas-cook-screens.png';
import JoyentScreenshots from '../../../assets/joyent-screens.png';
import KingfisherScreenshots from '../../../assets/kinfisher-screens.png';

module.exports = [
  {
    name: 'Trainline',
    objective: 'A Platform update & Improved User Experience',
    imagePath: TrainlineScreenshots,
    imageAlt: 'Trainline screenshots',
    headlineBgColour: '#56c0a8',
    headlineTextColour: '#ffffff',
    tagline:
      'Europe’s independent leader in the sale of train and coach tickets and the number one ticket reseller in the UK',
    challenges: [
      'Trainline were looking for a partner to onboard node as part an ongoing project to rebuild their online web experience.',
      'About 8 million customers a month come to the website and mobile app to purchase tickets. As part of the ongoing project to rebuild their online web experience, Trainline were looking for help on-boarding node and building up new skill sets and capabilities within their team to create a world class development organisation able to meet new demands.',
    ],
    results: [
      'Trainline had a legacy platform in urgent need of an updated interface, YLD wrapped the legacy platform in a Node.js layer that’s efficient, and most of all easy to use. Working in an integrated way with the team to build up node capabilities, YLD restructured the shopping basket and payment page to make it more user-friendly.',
      'Trainline have benefited from a significant increase in conversion rates on the site and an upskilled team with an entirely new skill-set and capabilities.',
    ],
    video: true,
    videoPath: 'placeholder :)',
  },
  {
    name: 'The Economist',
    objective: 'Website Redesign & Improved User Experience',
    imagePath: TheEconomistScreenshots,
    imageAlt: 'The Economist screenshots',
    headlineBgColour: '#383e42',
    headlineTextColour: '#ffffff',
    tagline:
      'A world renowned international current affairs, politics and news magazine',
    challenges: [
      'Technology is a core part of the way the business is run and an integral part of the storytelling industry, but the old The Economist website was hard to maintain and not a mobile first experience and in need of a redesign. The Economist wanted to improve their architecture and work with node and React to achieve a more flexible solution.',
      'The Economist needed a redesign and implementation of existing blogs sites with server-side rendering. In addition to this they needed to divert web traffic from a legacy CMS to a new set of micro-services.',
    ],
    results: [
      'YLD partnered with The Economist team to build a vision and introduced a team of dedicated and experienced engineers working together to seamlessly create a better reading experience. YLD helped construct a NodeJS and ReactJS system that decreased visitors’ perceived webpage load times. Over 12 months, YLD replaced all Economist.com user facing pages, replacing every piece of Html and using Node and React technologies to update the systems and the way they talk to each and deliver a content centric approach and more flexible solution.  Today, shared codebase reduces front- and back-end maintenance cost. Integration with new RESTful micro service tier resulted in faster response times requiring less hardware.',
    ],
    video: true,
    videoPath: 'placeholder :)',
  },
  {
    name: 'Thomas Cook',
    objective: 'Faster Release Cycles & Improved User Experience',
    imagePath: ThomasCookScreenshots,
    imageAlt: 'Thomas Cook screenshots',
    headlineBgColour: '#ffb92d',
    headlineTextColour: '#013a5c',
    tagline:
      'The Thomas Cook Group is the oldest and best known name in leisure travel with a history of innovation that started in 1841. Now serving 19 million customers, Thomas Cook is focused on transforming their business to deliver better their holiday dreams.',
    challenges: [
      'The Thomas Cook Group is the oldest and best known name in leisure travel with a history of innovation that started in 1841. Now serving 19 million customers, Thomas Cook is focused on transforming their business to deliver better their holiday dreams.',
    ],
    results: [
      'Some of the key results achieved were faster release cycles and quicker turnaround of new features, with improved performance and speed. From a mobile user perspective, there was a uplift of 30%, of which 20% was due to technology and responsive design for mobile.  ',
    ],
    video: true,
    videoPath: 'placeholder :)',
  },
  {
    name: 'Joyent',
    objective: 'xxx',
    imagePath: JoyentScreenshots,
    imageAlt: 'Joyent screenshots',
    headlineBgColour: '#3b4aaf',
    headlineTextColour: '#ffffff',
    tagline: 'xxx',
    challenges: ['xxx'],
    results: ['xxx'],
    video: false,
  },
  {
    name: 'Kingfisher',
    objective: 'Integrating new services & in-store mobile app',
    imagePath: KingfisherScreenshots,
    imageAlt: 'Kingfisher screenshots',
    headlineBgColour: '#3c3c3c',
    headlineTextColour: '#ffffff',
    tagline:
      'Kingfisher plc is an international home improvement company offering DIY and home improvement products and services.',
    challenges: [
      'The company needed an overhaul of their middle layer service that integrates the KF services and their C&C project (the mobile app for Kingfisher stores)',
      'Required expert micro services',
      'Creation of an internal NodeJS team',
    ],
    results: [
      'Using the best practises YLD helped build secure, scalable, and fast RESTful services.',
      'YLD used pair programming to upskill KF development team, to help them continue to implement new changes even after the contract term ended. ',
    ],
    video: false,
  },
];

// Object template
// {
//   name: '',
//   objective: '',
//   imagePath: '',
// imageAlt: 'xxx screenshots',
// headlineBgColour: ,
// headlineTextColour:,
//   tagline:
//     '',
//   challenges: [
//   ],
//   results: [
//   ],
//   video: true / false,
//   videoPath: 'placeholder :)',
// },
