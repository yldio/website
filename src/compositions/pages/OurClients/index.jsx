import React, { Fragment } from 'react';
import PageHero from 'components/PageHero';
import Container from 'components/Container';
import CaseStudyItem from '../../../components/CaseStudyItem';

import { Section } from './styled';
import ClientsLogos from './ClientsLogos';

const caseStudiesArray = require('./case-studies');

const OurClients = () => (
  <Fragment>
    <Section data-selector="our-clients:hero">
      <Container>
        <PageHero before="Our Clients" white big>
          Weâ€™re proud to work with some amazing clients who are as passionate
          about their industries as we are about helping them build their
          technology futures.
        </PageHero>
      </Container>
    </Section>
    <ClientsLogos />
    {caseStudiesArray.map(item => (
      <CaseStudyItem
        key={item.name}
        name={item.name}
        objective={item.objective}
        imagePath={item.imagePath}
        imageAlt={item.imageAlt}
        headlineBgColour={item.headlineBgColour}
        headlineTextColour={item.headlineTextColour}
        tagline={item.tagline}
        challenges={item.challenges}
        results={item.results}
        video={item.video}
        videoPath={item.videoPath}
      />
    ))}
  </Fragment>
);

export default OurClients;
