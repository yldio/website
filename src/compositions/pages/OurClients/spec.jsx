import React from 'react';
import { shallow } from 'enzyme';
import OurClients from './index';

test('compositions/pages/OurClients snapshot', () => {
  const wrapper = shallow(<OurClients />);

  expect(wrapper).toMatchSnapshot();
});
