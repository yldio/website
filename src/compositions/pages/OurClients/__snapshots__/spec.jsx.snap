// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/pages/OurClients snapshot 1`] = `
<React.Fragment>
  <styled__Section
    data-selector="our-clients:hero"
  >
    <Container>
      <PageHero
        before="Our Clients"
        big={true}
        white={true}
      >
        We’re proud to work with some amazing clients who are as passionate about their industries as we are about helping them build their technology futures.
      </PageHero>
    </Container>
  </styled__Section>
  <ClientsLogos />
  <CaseStudyItem
    challenges={
      Array [
        "Trainline were looking for a partner to onboard node as part an ongoing project to rebuild their online web experience.",
        "About 8 million customers a month come to the website and mobile app to purchase tickets. As part of the ongoing project to rebuild their online web experience, Trainline were looking for help on-boarding node and building up new skill sets and capabilities within their team to create a world class development organisation able to meet new demands.",
      ]
    }
    headlineBgColour="#56c0a8"
    headlineTextColour="#ffffff"
    imageAlt="Trainline screenshots"
    imagePath="/src/assets/trainline-screen.png"
    key="Trainline"
    name="Trainline"
    objective="A Platform update & Improved User Experience"
    results={
      Array [
        "Trainline had a legacy platform in urgent need of an updated interface, YLD wrapped the legacy platform in a Node.js layer that’s efficient, and most of all easy to use. Working in an integrated way with the team to build up node capabilities, YLD restructured the shopping basket and payment page to make it more user-friendly.",
        "Trainline have benefited from a significant increase in conversion rates on the site and an upskilled team with an entirely new skill-set and capabilities.",
      ]
    }
    tagline="Europe’s independent leader in the sale of train and coach tickets and the number one ticket reseller in the UK"
    video={true}
    videoPath="placeholder :)"
  />
  <CaseStudyItem
    challenges={
      Array [
        "Technology is a core part of the way the business is run and an integral part of the storytelling industry, but the old The Economist website was hard to maintain and not a mobile first experience and in need of a redesign. The Economist wanted to improve their architecture and work with node and React to achieve a more flexible solution.",
        "The Economist needed a redesign and implementation of existing blogs sites with server-side rendering. In addition to this they needed to divert web traffic from a legacy CMS to a new set of micro-services.",
      ]
    }
    headlineBgColour="#383e42"
    headlineTextColour="#ffffff"
    imageAlt="The Economist screenshots"
    imagePath="/src/assets/economist-screens.png"
    key="The Economist"
    name="The Economist"
    objective="Website Redesign & Improved User Experience"
    results={
      Array [
        "YLD partnered with The Economist team to build a vision and introduced a team of dedicated and experienced engineers working together to seamlessly create a better reading experience. YLD helped construct a NodeJS and ReactJS system that decreased visitors’ perceived webpage load times. Over 12 months, YLD replaced all Economist.com user facing pages, replacing every piece of Html and using Node and React technologies to update the systems and the way they talk to each and deliver a content centric approach and more flexible solution.  Today, shared codebase reduces front- and back-end maintenance cost. Integration with new RESTful micro service tier resulted in faster response times requiring less hardware.",
      ]
    }
    tagline="A world renowned international current affairs, politics and news magazine"
    video={true}
    videoPath="placeholder :)"
  />
  <CaseStudyItem
    challenges={
      Array [
        "The Thomas Cook Group is the oldest and best known name in leisure travel with a history of innovation that started in 1841. Now serving 19 million customers, Thomas Cook is focused on transforming their business to deliver better their holiday dreams.",
      ]
    }
    headlineBgColour="#ffb92d"
    headlineTextColour="#013a5c"
    imageAlt="Thomas Cook screenshots"
    imagePath="/src/assets/thomas-cook-screens.png"
    key="Thomas Cook"
    name="Thomas Cook"
    objective="Faster Release Cycles & Improved User Experience"
    results={
      Array [
        "Some of the key results achieved were faster release cycles and quicker turnaround of new features, with improved performance and speed. From a mobile user perspective, there was a uplift of 30%, of which 20% was due to technology and responsive design for mobile.  ",
      ]
    }
    tagline="The Thomas Cook Group is the oldest and best known name in leisure travel with a history of innovation that started in 1841. Now serving 19 million customers, Thomas Cook is focused on transforming their business to deliver better their holiday dreams."
    video={true}
    videoPath="placeholder :)"
  />
  <CaseStudyItem
    challenges={
      Array [
        "xxx",
      ]
    }
    headlineBgColour="#3b4aaf"
    headlineTextColour="#ffffff"
    imageAlt="Joyent screenshots"
    imagePath="/src/assets/joyent-screens.png"
    key="Joyent"
    name="Joyent"
    objective="xxx"
    results={
      Array [
        "xxx",
      ]
    }
    tagline="xxx"
    video={false}
    videoPath=""
  />
  <CaseStudyItem
    challenges={
      Array [
        "The company needed an overhaul of their middle layer service that integrates the KF services and their C&C project (the mobile app for Kingfisher stores)",
        "Required expert micro services",
        "Creation of an internal NodeJS team",
      ]
    }
    headlineBgColour="#3c3c3c"
    headlineTextColour="#ffffff"
    imageAlt="Kingfisher screenshots"
    imagePath="/src/assets/kinfisher-screens.png"
    key="Kingfisher"
    name="Kingfisher"
    objective="Integrating new services & in-store mobile app"
    results={
      Array [
        "Using the best practises YLD helped build secure, scalable, and fast RESTful services.",
        "YLD used pair programming to upskill KF development team, to help them continue to implement new changes even after the contract term ended. ",
      ]
    }
    tagline="Kingfisher plc is an international home improvement company offering DIY and home improvement products and services."
    video={false}
    videoPath=""
  />
</React.Fragment>
`;
